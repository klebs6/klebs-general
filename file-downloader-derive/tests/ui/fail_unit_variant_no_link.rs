
// file-downloader-derive/tests/ui/fail_unit_variant_no_link.rs
use file_downloader::FileDownloader;
use file_downloader_derive::FileDownloader;

#[derive(FileDownloader)]
enum MyEnum {
    // This variant has no link attribute
    VariantOne,
}

fn main() {}

