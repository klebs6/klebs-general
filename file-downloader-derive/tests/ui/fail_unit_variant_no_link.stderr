error: Missing `#[download_link = "..."]` attribute on this unit variant
 --> tests/ui/fail_unit_variant_no_link.rs:9:5
  |
9 |     VariantOne,
  |     ^^^^^^^^^^

error: `match` arm with no body
 --> tests/ui/fail_unit_variant_no_link.rs:9:5
  |
9 |     VariantOne,
  |     ^^^^^^^^^^- help: add a body after the pattern: `=> todo!(),`
